package.json

```diff
@@ -13,23 +13,21 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@material-ui/pickers": "^3.2.10",
-    "activedirectory2": "^1.3.0",
+    "@trieb.work/ldap-authentication": "^2.2.3",
```

Comment:
What's the reason for using a copy instead of the original package?

Reply:
as menationed below, this is just temporary until the newest version gets published to NPM.